test/new-e2e/runner/parameters/fixtures/config_no_aws_account.yaml

```diff
@@ -1,21 +1,10 @@
-# Environments configuration, validated and passed to Pulumi
```

Comment:
What do you think about adding `test` in the filename to make it explicit they are used in test?